DeusEx/Classes/DXRFixup.uc

```diff
@@ -1,12 +1,66 @@
 class DXRFixup expands DXRBase;
 
 function FirstEntry()
+{
+    Super.FirstEntry();
+    l( "mission " $ dxr.dxInfo.missionNumber $ " FirstEntry()");
+
+    Level.AmbientBrightness += dxr.flags.brightness;
+
+    if( dxr.dxInfo.missionNumber == 6 )
+        HongKong_FirstEntry();
+    if( dxr.dxInfo.missionNumber == 12 )
+        Vandenberg_FirstEntry();
+}
+
+function AnyEntry()
+{
+    Super.AnyEntry();
+    l( "mission " $ dxr.dxInfo.missionNumber $ " AnyEntry()");
+
+    FixUnbreakableCrates();
+
+    if( dxr.dxInfo.missionNumber == 6 )
+        HongKong_AnyEntry();
+}
+
+function FixUnbreakableCrates()
+{
+    local CrateUnbreakableLarge c;
+    foreach AllActors(class'CrateUnbreakableLarge', c) {
+        if( c.bInvincible ) {
+            c.bInvincible = False;
+            c.HitPoints = 2000;
```

Comment:
2000 health is enough where it would rarely be broken by accident, and these large crates are never required